### PR TITLE
Quick fix to work with new library updates

### PR DESCRIPTION
**Pull request by mario**
This is just so you can test, I will move it to 1.0.11 once you've tested it.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -41,7 +41,7 @@ dependencies {
     /// dependencies for app building
     compile name: 'touch-image-view'
 
-    compile 'com.github.nextcloud:android-library:1.0.10'
+    compile 'com.github.nextcloud:android-library:-SNAPSHOT'
```
Comment (AndyScherzinger) [inline build.gradle]: needs to be set to an actual release-version before merge

**Comment by AndyScherzinger** [comment]
👍 with setting a lib version before merge (will have to wait for @tobiasKaminsky's fix https://github.com/nextcloud/android-library/pull/33)

**Comment by AndyScherzinger** [comment]
👍 with setting a lib version before merge (will have to wait for @tobiasKaminsky's fix https://github.com/nextcloud/android-library/pull/33)

**Comment by AndyScherzinger** [comment]
👍 
@tobiasKaminsky please review and merge :)

**Comment by AndyScherzinger** [comment]
👍 
@tobiasKaminsky please review and merge :)

**Comment by tobiasKaminsky** [comment]
👍

**Pull request merged**